genre/conf/request-params.xml

```diff
@@ -0,0 +1,7 @@
+<lst name="defaults">
+  <str name="echoParams">explicit</str>
+  <str name="fl">score,_store</str>
+  <str name="qf">alias^1.75 genre^2 genreaccent^2.5 comment ngram^0.5</str>
+  <str name="pf">alias^1.5 genre^2 genreaccent^2.5 comment</str>
+  <str name="bf">log(sum(ref_count,1))^4</str>
```

Comment:
I have no idea, better to test it. Not merging for now, will give a try when other related PRs can be tested at the same time.

Reply:
I removed this for now since I 1) am not convinced we need to order genres by popularity 2) hadn't implemented the ref_count in sir to begin with :sweat_smile: